src/index.ts

```diff
@@ -187,18 +187,42 @@ export function declutter(arg: any): any {
   }
   let ret = { ...arg }
   // Marqeta can send back several empty fields on all responses... clean it
-  if (isEmpty(ret.nextUrl)) {
+  if (isEmpty(ret.metadata)) {
     delete ret.nextUrl
   }
-  if (isEmpty(ret.previousUrl)) {
-    delete ret.previousUrl
-  }
   if (ret.status === 200) {
     delete ret.status
   }
   return ret
 }
 
+/**
+ * Marqeta will return fields in a response that are invalid when used
+ * elsewhere in their API, e.g.: created_date, updated_date. These
+ * must be removed if using the json response elsewhere in Marqeta's API.
+ */
+export function cleanJson(arg: any): any {
+  // see if we have anything to do at all...
+  if (isEmpty(arg) || typeof arg !== 'object') {
+    return arg
+  }
+  let ret = { ...arg }
+  // Marqeta can send back several fields that cannot be used anywhere else...
+  if (ret?.created_time) {
+    delete ret.created_time
+  }
+  if (ret?.status) {
+    delete ret.status
+  }
+  if (isEmpty(ret?.metadata)) {
+    delete ret.metadata
+  }
+  if (ret?.last_modified_time) {
+    delete ret.last_modified_time
+  }
+  return ret
+}
```

Comment:
Don't need this because of a few things:

* Most JavaScript devs will know about destructuring to remove values
* The removal of just these three fields is likely specific to the update Business, but what about update Card, or update User... what will they do for `cleanJson()`?

Best to deal with this kind of issue right at the point of usage...

Reply:
> and does Marqeta care about empty arrays... and empty strings... or is it just empty Objects... and is it really only the metadata?

Marqeta ignored `identifications:[]` , so it doesn't care about empty arrays. Nor does Marqeta care about empty strings.  It does care about empty objects, e.g. `metdata`, `primary_contact`, `incorporation`, etc.